evals/AGENTS.md

```
@@ -0,0 +1,114 @@
+|--------|---------|------|
+| hello-world | 30s | $0.50 |
+| code-review | 60s | $0.50 |
+| jira/ready-to-solve | 3min | $2.00 |
+| jira/solve | 2min | $1.00 |
+
+### LLM judge
+The grading model (`defaultTest.options.provider`) is `vertex:claude-opus-4-6` — same model as the agent target. This is only invoked for `llm-rubric` assertions. Tests using only `icontains`, `skill-used`, `cost`, or `latency` don't call the judge.
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)